docs/releasing-files.md

```diff
@@ -10,6 +10,9 @@ The study author should do everything they can to make this easy; for example, c
 
 ## Releasing output files from analysis runs
 
+!!! note "The following information is for OpenSAFELY's internal output-checkers only"
+    Researchers MUST NOT release their output files themselves - contact disclosurecontrol@opensafely.org to make an release request
```

Comment:
nit: missing a full stop at the end.

Reply:
i'm trying to figure out how to re-edit :) but i can face it!